### ISSUE TITLE
Pandas 0.20 compatibility

### ISSUE THREAD

**Issue by sehnem** [tomchor/pymicra#2]
Great project!

I would like to know the main issues with pandas 0.20. I tested the package and was not able to find any.
I would also like to see this package on pip. It would be great to python newcomers.

**Comment by tomchor** [comment]
Thanks a lot! I did my best :)

You're probably talking about the `pandas==0.17` line on `setup.py`, I assume?

I put that in because when `pandas 0.18` got announced they said they had changed the API for `rolling_mean()` and were deprecating the old syntax. Since I didn't have time to test I figured I'd better be safe and fix the `pandas` version.

I now believe that the old syntax still works so that line is actually unnecessary. It needs to be taken out (I had completely forgotten about that). Although it would be good to migrate the `rolling_mean()` and `rolling_median()`s to the new syntax before.

Did you really test with `0.20` and it worked? If you did I'll take it out.

About `pip`, I totally agree. I plan to move the package to `pip` later this year when I have a bit more time to look into it. I'm presenting pymicra in a conference so it'll probably be after that.
